src/lib/metaphysics.ts

```diff
@@ -0,0 +1,60 @@
+interface Args {
```

Comment:
Rather than `lib` (which is somewhat foreign at artsy these days), maybe follow some of our other recent JS patterns where we make these types of things first class in a `system` directory. 

So 
- `system/metaphysics.ts` (or `system/network` -- ideally, later, it's `system/relay`, which is what we see everywhere now) 
- `system/tools`

Reply:
Maybe, top-level `tools` makes sense, esp given the other comment below.

Reply:
Also in https://github.com/artsy/quantum/pull/14/commits/fab86c8954b80941e3cb3cfb5ebc1e8737ef1e60